src/components/PanelMenuWrapper.js

```diff
@@ -51,13 +51,13 @@ class PanelsWithSidebar extends Component {
     const sections = [];
     const groupLookup = {};
     let groupIndex;
-    const panels = React.Children.toArray(children);
+    let childrenArray = React.Children.toArray(children);
```

Comment:
could be a ternary... let's try to avoid using `let` as much as we can please in new code :)

Reply:
in fact, let's go one more step: just always call `sortMenu` and if `menuPanelOrder` is null just early-return. much easier to read.